common/web/lm-worker/src/main/model-compositor.ts

```diff
@@ -410,6 +405,8 @@ export default class ModelCompositor {
     const keyedPrefix = keyed(truePrefix);
     const lowercasedPrefix = keyCased(truePrefix);
 
+    let deduplicatedSuggestionTuples: CorrectionPredictionTuple[] = [];
```

Comment:
```suggestion
    const deduplicatedSuggestions: CorrectionPredictionTuple[] = [];
```

We don't need to use `Tuple` in the variable name. Also should be `const` as I don't think we ever re-assign it later.

Note: renaming this variable made it much harder for me to follow the refactor!

Reply:
> Also should be const as I don't think we ever re-assign it later.

While true for _this_ PR, it was being re-assigned in #11900... but unnecessarily, since `.sort()` on an array can be done in place.  I'll make the `const` change there.

As the new name kinda got propagated around during all the ongoing work, I'd rather leave the new name in place (due to lots of merge conflicts that would likely result).  If necessary, we can always do a nomenclature clean-up pass later, once all the dust starts to settle.